steps/create_image.cwl

```diff
@@ -0,0 +1,151 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  s: 'http://schema.org/'
+  sbg: 'https://www.sevenbridges.com'
+id: create_image
+
+hints:
+  DockerRequirement:
+      dockerImageId: kernsuite/prefactor
+
+baseCommand: [python]
+inputs:
+  - id: mapfile_deep_low_size
+    type: File
+  - id: mapfile_nchansout_clean1
+    type: File
+  - id: mapfile_nwavelengths_low
+    type: File
+  - id: image_padding
+    type: float
+  - id: maxlambda_lowres
+    type: int
+  - id: cellsize_lowres_deg
+    type: float
+  - id: numcpu
+    type: int
+  - id: mem_pct
+    type: int
+  - id: prepcals
+    type: 'Directory[]'
+    inputBinding:
+      position: 19
+outputs:
+  - id: psf
+    type: File
+    outputBinding:
+      glob: wsclean-MFS-psf.fits
+  - id: dirty
+    type: File
+    outputBinding:
+      glob: wsclean-MFS-dirty.fits
+  - id: residual
+    type: File
+    outputBinding:
+      glob: wsclean-MFS-residual.fits
+  - id: model
+    type: File
+    outputBinding:
+      glob: wsclean-MFS-model.fits
+label: create_image
+arguments:
+  - position: 0
```

Comment:
When there are no `inputBindings` in `inputs` then the ordering of the command line arguments is very simple: it can be the order of entries under `arguments` if we don't have any `position` entries. So we can remove `position` to improve readability.